packages/api-sdk/package.json

```diff
@@ -7,20 +7,18 @@
     "url": "https://github.com/voiceflow/libs/issues"
   },
   "dependencies": {
-    "@types/chai": "^4.2.11",
-    "@types/jsonwebtoken": "^8.5.0",
     "@voiceflow/common": "^6.9.3",
-    "atob": "^2.1.2",
     "axios": "^0.21.1",
+    "jsonwebtoken": "^8.5.1",
     "superstruct": "^0.10.12"
   },
   "devDependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
-    "@types/atob": "^2.1.2",
+    "@types/chai": "^4.2.11",
+    "@types/jsonwebtoken": "^8.5.0",
```

Comment:
if a package is in your dependencies, the `@types` for it should be under `dependencies` as well. otherwise your library won't compile until users do it themselves. typescript themselves also say to do this:

> Here, our package depends on the `browserify` and typescript packages. `browserify` does not bundle its declaration files with its npm packages, so we needed to depend on `@types/browserify` for its declarations. `typescript`, on the other hand, packages its declaration files, so there was no need for any additional dependencies.

[src](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies)